### ISSUE TITLE
SVT: [VS Code] Some links are broken on the VS Code homepage

### ISSUE THREAD

**Issue by sujeilyfonseca** [eclipse/codewind#2131]
**Codewind version:** 0.9.0
**OS:** Mac OS Mojave 10.14.6 & VM with Windows 10

**IDE extension version:** 0.9.0
**IDE version:** 1.42.0

```
# Build date: 2020-02-11-14:47:19
build_info.url=https://download.eclipse.org/codewind/codewind-vscode/0.9.0/12
build_info.theia.SHA-1=f6532a10189552738d42ae1b98e89c180ad82a2a
build_info.SHA-1=0f2ab5a3f5d75b930b3dc8e6db26bb36148fac4e
```

**Description:** Some links are broken on the VS Code homepage

1. Locally: https://www.eclipse.org/codewind/mdt-vsc-firstproject.html
2. Remotely & Deploying Codewind Remotely: https://www.eclipse.org/codewind/remotedeploy-combo.html
3. Performance Monitoring: https://www.eclipse.org/codewind/performance.html

<img width="1282" alt="Screen Shot 2020-02-11 at 10 46 55 AM" src="https://user-images.githubusercontent.com/49531960/74252554-dbe47800-4cbb-11ea-9ef7-8c45875fca8a.png">

![Screen Shot 2020-02-11 at 10 42 11 AM](https://user-images.githubusercontent.com/49531960/74252926-64631880-4cbc-11ea-9a27-c90d4e6b6ec6.png)

@jagraj

**Comment by tetchel** [comment]
These pages won't be published until 0.9 release, they exist on the docs staging server

**Comment by tetchel** [comment]
https://github.com/eclipse/codewind/issues/2057
